### ISSUE TITLE
Document that the remote name should be origin

### ISSUE THREAD

**Issue by denisrosset** [lostintangent/gitdoc#54]
Hi,
the remote name is hard coded to be origin and I just have been bitten by this.
Should I make a PR against the Readme to document it, or would you like me to make it a configuration option?
I'd be happy to contribute.
Best,
Denis

**Comment by lostintangent** [comment]
Good catch! In addition to adding a config setting, I wonder if it also makes sense to just check and see if there's only one remote, and if so, just using that?

In either case, a PR would be very appreciated 🙏
